olm_deploy/scripts/catalog-deploy.sh

```diff
@@ -9,15 +9,6 @@ export IMAGE_LOGGING_KIBANA6=${IMAGE_LOGGING_KIBANA6:-registry.ci.openshift.org/
 export IMAGE_OAUTH_PROXY=${IMAGE_OAUTH_PROXY:-registry.ci.openshift.org/ocp/${OPENSHIFT_VERSION}:oauth-proxy}
 ELASTICSEARCH_OPERATOR_NAMESPACE=${ELASTICSEARCH_OPERATOR_NAMESPACE:-openshift-operators-redhat}
 
-if [ -n "${IMAGE_FORMAT:-}" ] ; then
```

Comment:
need to change `ocp` to `logging` in above env variables also?

Reply:
not for CI, but i will change it here too